server/server/views/home.py

```diff
@@ -1,5 +1,4 @@
 from cornice.resource import resource, view
```

Comment:
We don't have this file in views and we don't need it right now. Please remove this.